index.html

```diff
@@ -76,212 +77,13 @@ <h3 class="connect">LET'S CONNECT</h3>
-                </p>
-
-                <ul class="languages">
-                  <li>html</li>
-                  <li>Ruby on rails</li>
-                  <li>CSS</li>
-                  <li>JavaScript</li>
-                </ul>
-                <button class="beg-btn"><a href="#">
-                  See Project</a></button>
-            </div>
-          </div>
-
-          <!--Card Three-->
-          <div class="first-card" >
-            <div class="img-container">
-                <img class="card-img" src="./images/Snapshoot Portfolio (1).png" alt="picture card">
-            </div>
-            <div class="details-container">
-                <div class="heading">
-                  <h1>Facebook 360</h1>
-                </div>
-
-                <div class="details">
-                  <h3>FACEBOOK</h3>
-                  <img src="./images/Counter.png" alt="An image of a dot">
-                  <h4>Full Stack Dev</h4>
-                  <img src="./images/Counter.png" alt="An image of a dot">
-                  <h4>2015</h4>
-                </div>
-
-                <p>
-                  Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
-                </p>
-
-                <ul class="languages">
-                  <li>html</li>
-                  <li>Ruby on rails</li>
-                  <li>CSS</li>
-                  <li>JavaScript</li>
-                </ul>
-                <button class="beg-btn"><a href="#">
-                  See Project</a></button>
-            </div>
-          </div>
-
-          <!--Card Four-->
-          <div class="first-card" >
-            <div class="img-container">
-                <img class="card-img" src="./images/Snapshoot Portfolio (3).png" alt="picture card">
-            </div>
-            <div class="details-container">
-                <div class="heading">
-                  <h1>Uber Navigation</h1>
-                </div>
-
-                <div class="details">
-                  <h3>Uber</h3>
-                  <img src="./images/Counter.png" alt="An image of a dot">
-                  <h4>Lead Developer</h4>
-                  <img src="./images/Counter.png" alt="An image of a dot">
-                  <h4>2018</h4>
-                </div>
-
-                <p>
-                  A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer:your car.
-                </p>
-
-                <ul class="languages">
-                  <li>html</li>
-                  <li>Ruby on rails</li>
-                  <li>CSS</li>
-                  <li>JavaScript</li>
-                </ul>
-                <button class="beg-btn"><a href="#">
-                  See Project</a></button>
-            </div>
-          </div>
+        <section class="section-1" id="Portfolio">
         </section>
```

Comment:
- [ ] Good job implementing the pop-up window milestone 👍 , but I'm not about to see the projects in your page, kindly fix that 

https://user-images.githubusercontent.com/80458199/233792675-3028a7bc-ab55-4624-b55a-60efb3ce9a73.mp4

Reply:
I fixed my error and i want to request another review but my dashboard says review pending